src/test/kotlin/io/sip3/commons/domain/payload/RtpReportPayloadTest.kt

```diff
@@ -0,0 +1,193 @@
+            assertEquals(ssrc, byteBuf.readLong())
+
+            assertEquals(RtpReportPayload.TAG_CALL_ID, byteBuf.readByte().toInt())
+            assertEquals(CALL_ID_VALUE_LENGTH, byteBuf.readShort().toInt())
+            val actualBytes = ByteArray(CALL_ID.length)
+            byteBuf.readBytes(actualBytes)
+            val decodedString = actualBytes.toString(Charset.defaultCharset())
+            assertEquals(callId, decodedString)
+
+            assertEquals(RtpReportPayload.TAG_EXPECTED_PACKET_COUNT, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(expectedPacketCount, byteBuf.readInt())
+
+            assertEquals(RtpReportPayload.TAG_RECEIVED_PACKET_COUNT, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(receivedPacketCount, byteBuf.readInt())
+
+            assertEquals(RtpReportPayload.TAG_LOST_PACKET_COUNT, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(lostPacketCount, byteBuf.readInt())
+
+            assertEquals(RtpReportPayload.TAG_REJECTED_PACKET_COUNT, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(rejectedPacketCount, byteBuf.readInt())
+
+            assertEquals(RtpReportPayload.TAG_DURATION, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(duration, byteBuf.readInt())
+
+            assertEquals(RtpReportPayload.TAG_LAST_JITTER, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(lastJitter, byteBuf.readFloat())
+
+            assertEquals(RtpReportPayload.TAG_AVG_JITTER, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(avgJitter, byteBuf.readFloat())
+
+            assertEquals(RtpReportPayload.TAG_MIN_JITTER, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(minJitter, byteBuf.readFloat())
+
+            assertEquals(RtpReportPayload.TAG_MAX_JITTER, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(maxJitter, byteBuf.readFloat())
+
+            assertEquals(RtpReportPayload.TAG_R_FACTOR, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(rFactor, byteBuf.readFloat())
+
+            assertEquals(RtpReportPayload.TAG_MOS, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(mos, byteBuf.readFloat())
+
+            assertEquals(RtpReportPayload.TAG_FRACTION_LOST, byteBuf.readByte().toInt())
+            assertEquals(7, byteBuf.readShort())
+            assertEquals(fractionLost, byteBuf.readFloat())
+        }
+    }
+
+    private fun getRtpReportPayload(callId: String?): RtpReportPayload {
```

Comment:
If we will have call_id as `lateinit var` - let's move this object to companion. All global class constants should stay there.